clang/test/CIR/CodeGen/CUDA/texture.cu

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: x86-registered-target
+// REQUIRES: nvptx-registered-target
+
+// RUN: %clang_cc1 -fclangir -std=c++11 -fcuda-is-device -triple nvptx64-nvidia-cuda -emit-llvm -o - %s | FileCheck --check-prefix=DEVICE %s
+// RUN: echo "GPU binary would be here" > %t
+// RUN: %clang_cc1 -fclangir -std=c++11 -triple x86_64-unknown-linux-gnu -target-sdk-version=8.0 -fcuda-include-gpubinary %t -emit-llvm -o - %s | FileCheck --check-prefix=HOST %s
+
+// Accessing nvvm intrinsics in this way no longer works.
+// XFAIL: *
```

Comment:
Note sure what you comment means. What's missing to un-XFAIL this test? All PRs that implement feature need a testcase or a concrete plan follow up plan.